### PR TITLE
feat: Run tests for sub packages in CI

### PR DESCRIPTION
**Pull request by zashraf1985**
## Summary
Travis CI Only run tests for `optimizely-sdk` package. This PR makes Travis CI run unit tests for other sub packages on `node 12`

## Test Plan
Tested on Travis CI

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -52,3 +53,14 @@ jobs:
     - stage: Cross-browser and umd unit tests
       node_js: '8'
       script: npm run test-ci
+    - &packagetest
+      stage: 'Test sub packages'
+      node_js: '12'
+      before_install: cd packages/datafile-manager
+      before_script: tsc
```
Comment (mjc1283) [inline .travis.yml]: Is this `tsc` necessary?

File: packages/utils/src/index.ts [hunk 2]
```diff
@@ -62,7 +62,7 @@ export function objectValues<K>(obj: { [key: string]: K }): K[] {
   return Object.keys(obj).map(key => obj[key])
 }
 
-export function objectEntries<K>(obj: { [key: string]: K }): [string, K][] {
+export function objectEntries<K>(obj: { [key: string]: K }): (string | K)[][] {
```
Comment (mjc1283) [inline packages/utils/src/index.ts]: What is this change?

Comment (zashraf1985) [reply packages/utils/src/index.ts]: This is fixing a tsc compilation error. return type was not correct.

File: .travis.yml (hunk 1 above)
Comment (zashraf1985) [reply .travis.yml]: not needed. i just checked. looks like jest automatically compiles typescript before use

File: packages/logging/package.json [hunk 3]
```diff
@@ -42,6 +42,7 @@
   "devDependencies": {
     "@types/jest": "^23.3.12",
     "jest": "^23.6.0",
-    "ts-jest": "^23.10.5"
+    "ts-jest": "^23.10.5",
+    "typescript": "^3.7.3"
```
Comment (mjc1283) [inline packages/logging/package.json]: Please change typescript dependency versions to `"3.3.3333"` in all packages

**Pull request merged**